ocdaction/ocdaction/templates/core/think.html

```diff
@@ -3,11 +3,116 @@
 {% load static %}
 
 {% block main %}
-    <section class="container">
-    	<div class="row">
-	    	<div class="col-xs-12">
-				<h1>Think</h1>
-				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse ac finibus ligula, at imperdiet nibh. Donec laoreet est vitae leo pulvinar, in porta risus viverra. Proin mollis diam sit amet lacus imperdiet ornare eu nec erat. Etiam semper lectus eget mi molestie hendrerit. Morbi vitae ornare odio. Phasellus vestibulum et ex bibendum commodo. Ut interdum egestas nisl at vestibulum. Suspendisse blandit lorem eros, feugiat consequat nibh vestibulum in. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis velit ante, pretium et arcu a, rutrum sodales sapien. Fusce vitae purus lacinia, laoreet enim at, consequat odio.</p>
+    <section>
+    	<div class="container">
+	    	<div class="row">
+		    	<div class="col-xs-12 col-md-8">
+					<h1 class="h1">Think</h1>
```

Comment:
I think that makes better sense, I was confused myself why it was done this way. Sorry been very busy and away but I will allocate some time over the weekend to complete this :)